### ISSUE TITLE
Failed to activate the pomodoro package

### ISSUE THREAD

**Issue by alkan** [yoshiori/pomodoro#9]
[Enter steps to reproduce below:]

1. Packages > Pomodoro > Start
2. I gor error as follows:

Cannot read property 'prototype' of undefined
TypeError: Cannot read property 'prototype' of undefined
  at __extends (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:1:1)
  at /home/ayhan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:5:3
  at Object.<anonymous> (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:46:5)
  at Object.<anonymous> (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/usr/share/atom/resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro.coffee:3:16)
  at Object.<anonymous> (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/usr/share/atom/resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)

**Atom Version**: 0.210.0
**System**: Ubuntu 15.04
**Thrown From**: [pomodoro](https://github.com/yoshiori/pomodoro) package, v0.4.3

### Stack Trace

Failed to activate the pomodoro package

```
At Cannot read property 'prototype' of undefined

TypeError: Cannot read property 'prototype' of undefined
  at __extends (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:1:1)
  at /home/ayhan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:5:3
  at Object.<anonymous> (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:46:5)
  at Object.<anonymous> (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro-view.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/usr/share/atom/resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro.coffee:3:16)
  at Object.<anonymous> (/home/ayhan/.atom/packages/pomodoro/lib/pomodoro.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/usr/share/atom/resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Package.module.exports.Package.requireMainModule (/usr/share/atom/resources/app.asar/src/package.js:709:34)
  at Package.module.exports.Package.activateConfig (/usr/share/atom/resources/app.asar/src/package.js:257:12)
  at Package.module.exports.Package.activateNow (/usr/share/atom/resources/app.asar/src/package.js:236:14)
  at /usr/share/atom/resources/app.asar/src/package.js:772:25
  at Emitter.module.exports.Emitter.emit (/usr/share/atom/resources/app.asar/node_modules/event-kit/lib/emitter.js:82:11)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:219:20)
  at CommandRegistry.handleCommandEvent (/usr/share/atom/resources/app.asar/src/command-registry.js:3:61)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/usr/share/atom/resources/app.asar/src/command-registry.js:153:19)
  at EventEmitter.<anonymous> (/usr/share/atom/resources/app.asar/src/window-event-handler.js:70:30)
  at emitOne (events.js:77:13)
  at EventEmitter.emit (events.js:166:7)

```

### Commands

```
     -8:53.5.0 pane:reopen-closed-item (atom-text-editor.editor.is-focused)
  5x -1:05.2.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:52 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:21.3.0 core:close (atom-workspace.workspace.scrollbars-visible-always.theme-atom-dark-syntax.theme-atom-dark-ui)
     -0:00.8.0 pomodoro:start (atom-workspace.workspace.scrollbars-visible-always.theme-atom-dark-syntax.theme-atom-dark-ui)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
language-vue-component, v0.2.0
pomodoro, v0.4.3

# Dev
No dev packages
```

**Comment by aleksandar-todorovic** [comment]
`pomodoro` package still contains references to functions that became deprecated and removed from Atom on June 1st.

I'm testing the solution right now, will create a pull request if everything works.

**Comment by yoshiori** [comment]
fix 90a0a0898a3c41293e3f23a3b0f0cdf5acba436f
